client/src/containers/CreateCardFormContainer/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import CreateCardForm from 'src/components/CreateCardForm'
+
+// import styles from './styles.css';
+
+class CreateCardFormContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: '',
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  render = () => (
+    <CreateCardForm
+      description={this.description}
+      handleChange={this.handleChange}
+      handleSubmit={this.handleSubmit}
+    />
+  );
+
+  handleChange(event) {
+    this.setState({ description: event.target.value })
+  }
+
+  handleSubmit(event) {
+    alert(`An card was submitted: ${this.state.description}`)
+    event.preventDefault()
+  }
+}
```

Comment:
@DDKatch thoughts about giving handleChange and handleSubmit names related to business logic?

* `updateDescription` instead of `handleChange`
* `createCard` instead of `handleSubmit`